src/ext/java/org/opentripplanner/ext/transmodelapi/model/plan/TripQuery.java

```diff
@@ -150,6 +151,14 @@ public static GraphQLFieldDefinition create(
           .type(new GraphQLNonNull(LocationInputType.INPUT_TYPE))
           .build()
       )
+      .argument(
+        GraphQLArgument
+          .newArgument()
+          .name("passthrough")
+          .description("The passthrough points as stops")
+          .type(new GraphQLList(new GraphQLNonNull(PassthroughPointInputType.INPUT_TYPE)))
+          .build()
+      )
```

Comment:
I think we are aiming for something like this:

```
# cardinality: [0-3] , missing, or empty -> no pass-through point
passThroughPoints : [PassThroughPointInput!] = []

type PassThroughPointInput {
  # Optional for logging/debugging only
  name : String


  # List of Quays, StopPlaces and/or MultiModalStopPlace ids
  placeIds : [String!]!
}
```

Reply:
Sorry for the detour with the more complex wrapper type, but this is good enough.